packages/axios-extension/src/abap/ui5-abap-repository-service.ts

```diff
@@ -198,8 +198,8 @@ export class Ui5AbapRepositoryService extends ODataService {
     public async undeploy({ bsp, testMode = false }: UndeployConfig): Promise<AxiosResponse | undefined> {
         const config = this.createConfig(bsp.transport, testMode);
         const host = this.getAbapFrontendUrl();
+        const info: AppInfo = await this.getInfo(bsp.name);
```

Comment:
This was moved out of the try/catch as the calling client will handle it and log the error if appropiate. Same approach as the deployment flow.